app/src/main/AndroidManifest.xml

```diff
@@ -73,7 +74,13 @@
             android:exported="false"
             android:theme="@style/AppTheme.Dialog" />
 
-        <service android:name=".service.WebSocketService" />
+        <service
+            android:name=".service.WebSocketService"
+            android:foregroundServiceType="specialUse">
+            <property
+                android:name="android.app.PROPERTY_SPECIAL_USE_FGS_SUBTYPE"
+                android:value="explanation_for_special_use" />
```

Comment:
What do you think about:
> This is the main feature of this application. The foreground notification displays the connectivity status to the configurable remote server and the service notifies the user when a new message has been published on the remote server.

Reply:
Sounds good

Reply:
Okay, then let's use this for now and see what google approve team will say about it (:.

Reply:
I hope this is the format they expect.  
Interesting enough that most examples I've found on GitHub have only defined a single word or a very short description:  
https://github.com/search?q=android.app.PROPERTY_SPECIAL_USE_FGS_SUBTYPE&type=code

Reply:
I guess more information doesn't really hurt. Are you sure that it's possible to use string references there? Most uses of special use only have plain text.

Reply:
Good question, let's instead just use a raw string in the manifest.

Reply:
Done in https://github.com/gotify/android/pull/325/commits/31162e435ac72bffeacd8c801f753fe742d156b4